app/controllers/reviewables_controller.rb

```diff
@@ -6,6 +6,7 @@ class ReviewablesController < ApplicationController
   PER_PAGE = 10
 
   before_action :version_required, only: [:update, :perform]
+  before_action :can_see_review_queue
```

Comment:
nitpick but I would call this something different than `can_` - something simple like `ensure_can_see` would be closer to the operation.